Sources/Fullscreen/ExploreDetail/Helpers/NSCollectionLayoutSection+Staggered.swift

```diff
@@ -93,9 +94,13 @@ public extension NSCollectionLayoutSection {
 
 private struct GridLayoutConfiguration {
     init(traitCollection: UITraitCollection) {
-        numberOfColumns = traitCollection.horizontalSizeClass == .regular ? 3 : 2
+        var columns = traitCollection.horizontalSizeClass == .regular ? 3 : 2
+        if (traitCollection.preferredContentSizeCategory.isAccessibilityCategory && Config.isDynamicTypeEnabled){
```

Comment:
Maybe this bool should be a new property on Config, something like isDynamicAccessibilityTypeEnabled? If it is likely this check will be done multiple times.

Reply:
Yes, that is a good idea! I will probably do more accessibility adjustments some time soon, and I will keep that in mind then